src/components/Profile/ProfileForm.js

```diff
@@ -174,14 +196,126 @@ class ProfileForm extends Component {
+                  </select>
+                </label>
+              </div>
+              <div className="form-group">
+                <label className="">
+                  How clean are you? &nbsp;
+                  <div className="form-check">
+                    <label>
+                      <input
+                        type="radio"
+                        name=""
+                        value="Clean"
+                        checked={addCleanliness === "Clean"}
+                        className="form-check-input"
+                        onChange={this.handleCleanlinessChange}
+                      />
+                      Clean
+                      </label>
+                  </div>
+
+                  <div className="form-check">
+                    <label>
+                      <input
+                        type="radio"
+                        name=""
+                        value="So-so"
+                        checked={addCleanliness === "So-so"}
+                        className="form-check-input"
+                        onChange={this.handleCleanlinessChange}
+                      />
+                      So-so
+                      </label>
+                  </div>
+
+                  <div className="form-check">
+                    <label>
+                      <input
+                        type="radio"
+                        name=""
+                        value="Messy"
+                        checked={addCleanliness === "Messy"}
+                        className="form-check-input"
+                        onChange={this.handleCleanlinessChange}
+                      />
+                      Messy
+                      </label>
+                  </div>
+                </label>
+              </div>
+              <div className="form-group">
+                <label className="">
+                  Do you wake up early? &nbsp;
+                  <div className="form-check">
+                    <label>
+                      <input
+                        type="radio"
+                        name=""
+                        value="Yes"
+                        checked={addWakeUp === "Yes"}
+                        className="form-check-input"
+                        onChange={this.handleWakeUpChange}
+                      />
+                      Yes
+                      </label>
+                  </div>
+
+                  <div className="form-check">
+                    <label>
+                      <input
+                        type="radio"
+                        name=""
+                        value="No"
+                        checked={addWakeUp === "No"}
+                        className="form-check-input"
+                        onChange={this.handleWakeUpChange}
+                      />
+                      No
+                      </label>
+                  </div>
+                </label>
+              </div>
 
```

Comment:
You could write a helper function that takes a question and and handle function and returns the question component so you don't have as much code for each of these. Would also make it a lot easier to add questions later